azurerm_application_insights_web_test/ping.xml

```diff
@@ -0,0 +1,6 @@
+<WebTest Name="${name}" Id="ABD48585-0831-40CB-9069-682EA6BB3583" Enabled="True" CssProjectStructure="" CssIteration="" Timeout="${timeout}" WorkItemIds=""
+    xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010" Description="" CredentialUserName="" CredentialPassword="" PreAuthenticate="True" Proxy="default" StopOnError="False" RecordedResultFile="" ResultsLocale="">
+    <Items>
+        <Request Method="GET" Guid="a5f10126-e4cd-570d-961c-cea43999a200" Version="1.1" Url="${url}" ThinkTime="0" Timeout="${timeout}" ParseDependentRequests="False" FollowRedirects="True" RecordResult="True" Cache="False" ResponseTimeGoal="0" Encoding="utf-8" ExpectedHttpStatusCode="200" ExpectedResponseUrl="" ReportingName="" IgnoreHttpStatusCode="False" />
```

Comment:
Why don't we put the expected status code as a parameter? 
Can we use this web test module to test an external service that returns a status code different than 200?
For instance a redirect?

Reply:
thank you @uolter 👍 I added http_method (default GET) and http_status_code (default 200).
Anyway it's possible to override ping.xml file for more specific web test.